mfa/methods/code/messaging.md

```diff
@@ -0,0 +1,146 @@
+{% method %}
+
+## Two-Factor Authentication with Messaging
+Two-Factor authentication with Bandwidth MEssaging services
+
+### Request URL
+
+<code class="post">POST</code>`https://mfa.bandwidth.com/api/v1/accounts/{accountId}/code/voice`
+
+#### Basic Authentication
+
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
+
+---
+
+### Supported Parameters
+
+| Parameter     | Description           |
+|:--------------|:----------------------|
+| to            | To telephone number   |
+| from          | From telephone number |
+| applicationId | Application Id        |
```

Comment:
overly indicate that "applicationId" for messaging 2fa should be a "messaging Application"

Reply:
Done 👍